lib/aggregate/attribute/hash.rb

```diff
@@ -74,10 +74,8 @@ def convert_to_json(value)
       def should_store_hash_as_json
         if options.key?(:store_hash_as_json)
           options[:store_hash_as_json]
-        elsif options.dig(:aggregate_db_storage_type) == :elasticsearch
-          false
         else
-          true
+          options.dig(:aggregate_db_storage_type) != :elasticsearch
```

Comment:
AFAICT `dig` should be deprecated in favor of `[key0][key1][key2]... rescue nil`.